phidgets_api/src/phidget.cpp

```diff
@@ -111,19 +113,49 @@ void Phidget::errorHandler(int error)
 int Phidget::AttachHandler(CPhidgetHandle handle, void *userptr)
 {
   ((Phidget*)userptr)->attachHandler();
+  is_connected = true;
+  updater.force_update();
```

Comment:
Move these two lines to the non-static function (`attachHandler`), won't compile otherwise.